examples/angular-notification-center-example/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "angular-notification-center-example",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.9.0",
```
What do you think? Should we align with our version number?

---

If the examples won't change on our different future versions indeed won't be needed.